terraform/api.tf

```diff
@@ -75,16 +74,15 @@ resource "aws_apigatewayv2_stage" "api" {
   name        = "$default"
   auto_deploy = true
 
-  access_log_settings {
+  /*access_log_settings {
```

Comment:
Let's just delete this if we aren't using it, we can always refer to history if we need it again

Reply:
Fixed